### PR TITLE
[doc] Updated doc for Textual Editor

### PR DESCRIPTION
**Pull request by georgiana-e**
Change-Id: Ifa98e3ceb611390be7532a0efe6e2c444af737a5
Signed-off-by: Georgiana-Elena ECOBICI <georgiana-elena.ecobici@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html [hunk 1]
```diff
@@ -23,8 +23,9 @@ <h3 id="Consistency_of_the_data">Consistency of the data</h3>
 			<img width="760" align="middle" border="0" src="Images/IS_System_SyncButtons.png"/>
 		</p>
 		<h2 id="Syntax_for_the_Textual_Scenario_Editor">Syntax for the Textual Scenario Editor</h2>
-		<h3 id="Adding_Instance_Roles">Adding Instance Roles</h3>
-		<p>Depending on the type of scenario (OES, OAS, IS, FS, ES) and the architecture level on which our diagram is located, we can add instance roles specifying a keyword (actor, component, function etc) and the name of the instance role (currently the parts associated need to exist already).</p>
+		<p>The syntax supported in the textual editor for IS, FS, ES Capella Scenarios is presented in the following paragraphs.</p>
+		<h3 id="Participants">Participants</h3>
+		<p>Depending on the type of scenario (OES, OAS, IS, FS, ES) and the architecture level on which our diagram is located, we can insert instance roles specifying a keyword (actor, component, function etc) and the name of the instance role (currently the parts associated need to exist already).</p>
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: need to already exist

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html [hunk 2]
```diff
@@ -91,6 +92,12 @@ <h3 id="Adding_Messages">Adding Messages</h3>
 					</li>
 				</ul>
 			</li>
+			<li>
+				<b> return branch </b>: to mark that a message has a return,the 
+				<b>withReturn</b> keyword can be used; if the message does not return immediately and other events ocur on its execution, then the
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: occur

Comment (claudiasima-ro) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: Replace "ocur" with "occur"

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki [hunk 3]
```diff
@@ -56,6 +58,7 @@ The user can create messages between participants assuming the fact that the exc
 ::: "A 1" -> "A 2" ''' withExecution ''' : "exec"
 ::: "A 2" -> "A 3"
 ::: '''deactivate''' "A 2"
+* ''' return branch ''': to mark that a message has a return,the '''withReturn''' keyword can be used; if the message does not return immediately and other events ocur on its execution, then the '''withExecution''' keyword can be used in the same time with the '''withReturn''' keyword to mark that the current message is a complex message and has return.
```
Comment (claudiasima-ro) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: ocur -> occur

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html [hunk 4]
```diff
@@ -133,6 +153,110 @@ <h4 id="Examples_2">Examples</h4>
...
+				</dl>
+			</li>
+			<li>for the other combined fragments: 
+				<b>PAR, LOOP, ASSERT, CONSIDER, CRITICAL, IGNORE, NEG, OPT, SEQ, STRICT, UNSET</b>':
+				<dl>
+					<dd>
+						<dl>
+							<dd>
+								<b>par</b> “condition A” 
+								<b>over</b> timelime1, timeline2,… {
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: typo: timeline

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html (hunk 1 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: ok

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki (hunk 3 above)
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: space after "," : has a return, the ....

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html (hunk 2 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: ok

Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.html]: ok

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki [hunk 5]
```diff
@@ -56,6 +58,7 @@ The user can create messages between participants assuming the fact that the exc
...
 * ''' create messages ''': it will add a create messages between the source and the target instance roles which shall already exist, otherwise a validation error is displayed.
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: a create message

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki [hunk 6]
```diff
@@ -74,4 +80,45 @@ The user can create messages between participants assuming the fact that the exc
...
+* for '''ALT''':
+::: '''alt''' �condition A� '''over''' timelime1, timeline2,� {
+::::[something]
+:::} '''else''' �condition B� {
+::::[something_else]
+:::} '''else''' �condition C� {
+::::[something_else]
+:::}
+* for the other combined fragments: '''PAR, LOOP, ASSERT, CONSIDER, CRITICAL, IGNORE, NEG, OPT, SEQ, STRICT, UNSET'''':
+::: '''par''' �condition A� '''over''' timelime1, timeline2,� {
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: timeline (typo)

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki (hunk 3 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki:61]: ???

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki (hunk 5 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: ??

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki (hunk 6 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: ok

File: plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki (hunk 3 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki:61]: has a return instead of has return?

Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki]: ok

**Review by MalinaStefaniaStoicanescu — APPROVED** [review]

**Review by georgiana-ecobici — APPROVED** [review]

**Pull request merged**